### PR TITLE
Support for customer portals

### PR DESCRIPTION
**Pull request by driesvints**

### CODE REVIEW

File: src/Concerns/ManagesCustomer.php [hunk 1]
```diff
@@ -64,4 +67,44 @@ public function lemonSqueezyTaxNumber(): ?string
...
     }
+
+    /**
+     * Get the customer portal url for this billable.
+     */
+    public function customerPortalUrl(): string
+    {
+        $this->assertCustomerExists();
+
+        $response = LemonSqueezy::api('GET', "customers/{$this->lemon_squeezy_id}");
```
Comment (dylanmichaelryan) [inline src/Concerns/ManagesCustomer.php:78]: Shouldn't this be

`$response = LemonSqueezy::api('GET', "customers/{$this->customer->lemon_squeezy_id}");`

File: src/Concerns/ManagesCustomer.php [hunk 2]
```diff
@@ -64,4 +67,44 @@ public function lemonSqueezyTaxNumber(): ?string
...
+    {
+        return new RedirectResponse($this->customerPortalUrl());
+    }
+
+    /**
+     * Determine if the billable has a Lemon Squeezy customer ID.
+     */
+    public function hasLemonSqueezyId(): bool
+    {
+        return ! is_null($this->lemon_squeezy_id);
```
Comment (dylanmichaelryan) [inline src/Concerns/ManagesCustomer.php:96]: Shouldn't this be

```return ! is_null($this->customer->lemon_squeezy_id);```

File: src/Concerns/ManagesCustomer.php (hunk 1 above)
Comment (driesvints) [reply src/Concerns/ManagesCustomer.php:78]: @dylanmichaelryan you're totally right. I've released a fix for this.

File: src/Concerns/ManagesCustomer.php (hunk 2 above)
Comment (driesvints) [reply src/Concerns/ManagesCustomer.php:96]: I removed this method again. Thanks!

**Comment by driesvints** [comment]
FYI: some post-merge fixes: https://github.com/lmsqueezy/laravel/pull/47

**Pull request merged**